packages/components/src/components/dropdown/dropdown.js

```diff
@@ -257,7 +261,9 @@ class Dropdown extends mixin(
       if (
         !current.matches(this.options.selectorItemHidden) &&
         !current.parentNode.matches(this.options.selectorItemHidden) &&
-        !current.matches(this.options.selectorItemSelected)
+        (isShowSelected ||
```

Comment:
Given navigating the selected item yields letting user select an item that is selected already - Probably we'd leave this logic here as-is?

Reply:
@asudoh in the React version you can navigate to the selected item (and then re-select it), why would be differ since this PR is to get vanilla closer to what React is doing.

Reply:
OK, thank you for clarifying!